src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -76,6 +76,9 @@ static RuntimeMetrics()
                 unit: "bytes",
                 description: "The amount of committed virtual memory for the managed GC heap, as observed during the latest garbage collection. Committed virtual memory may be larger than the heap size because it includes both memory for storing existing objects (the heap size) and some extra memory that is ready to handle newly allocated objects in the future. The value will be unavailable until garbage collection has occurred.");
 
```

Comment:
Can we add a comment and an issue tracking restoring the SizeAfterBytes code once this runs on .NET 7? Private reflection is fine for .NET 6 where we know it works, but it would be unfortunate to forget about it. Its possible in some future version of .NET we'll remove or rename the internal API and we wouldn't this code to suddenly break.

Reply:
I've added comment and an issue #496.

Reply:
@noahfalk Will this internal API ever be removed/renamed in future .NET 6 minor/patch versions, or just possibly in .NET 7 and future versions? If it's the later, then I can safely assume it's always available until we support .NET 7.

Reply:
I think its safe to assume that API will never go away in .NET 6 servicing versions.